client/src/components/ColorList.js

```diff
@@ -21,10 +22,28 @@ const ColorList = ({ colors, updateColors }) => {
     // Make a put request to save your updated color
     // think about where will you get the id from...
     // where is is saved right now?
+    withAuth().put('http://localhost:5000/api/colors/:id', colorToEdit)
```

Comment:
Forgot to make the ID dynamic? Otherwise changes are not being saved on back end. Make sure to check your app few times before thinking you're done with it. 👍